package.json

```diff
@@ -16,7 +16,8 @@
     "scripts": {
         "build-icons": "rimraf dist dist-es5 typings && mkdirp dist dist-es5 typings && node buildIcons",
         "build-example": "rimraf example && mkdirp example && babel-node --presets react,env buildExample",
-        "semantic-release": "semantic-release"
+        "semantic-release": "semantic-release",
+        "prepare": "npm run build-icons"
```

Comment:
This is the only actual change, to `npm run build-icons` before publishing and after installation:

> prepare: Run both BEFORE the package is packed and published, on local npm install without any arguments, and when installing git dependencies (See below). This is run AFTER prepublish, but BEFORE prepublishOnly. [Source](https://docs.npmjs.com/misc/scripts)